src/main/java/com/commerce/backendserver/global/exception/error/ErrorCode.java

```diff
@@ -0,0 +1,10 @@
+package com.commerce.backendserver.global.exception.error;
+
+import org.springframework.http.HttpStatus;
+
+public interface ErrorCode {
+
+  String getMessage();
+
+  HttpStatus getStatus();
+}
```

Comment:
최근 다른 프로젝트를 진행하면서 우리가 이용하던 사이트들의 에러 응답에 대해 고민해봤어요. 

<img width="573" alt="스크린샷 2023-08-09 오후 2 05 11" src="https://github.com/TeamCommerce/backend-server/assets/112257466/aad49f5e-14ee-4f8b-91ec-7a5c8f7b45cc">

우리가 흔히 이용하던 사이트에서 `사용자` 그리고 `개발자`가 그 에러를 구분할 수 있는 고유 에러코드가 같이 존재하고, 규칙을 정해서 관리하면, 에러코드를 전역에서 더 효율적으로 관리하고, 사용자 입장에서도 트러블 슈팅하기 좋을 것 같다는 생각이 드는데 어떻게 생각하시는지 궁금합니다.

```suggestion
public interface ErrorCode {

  String getMessage();
  String uniqueCode() // 가제;
  HttpStatus getStatus();
}
```